src/main/java/umk/mat/pajda/ProjektZespolowy/controllers/ReviewDataController.java

```diff
@@ -22,6 +24,11 @@
     name = "Review Endpoints",
     description = "Controller for handling requests related to add/del/patch/read reviews")
 public class ReviewDataController {
+  @Value("${app.isProd:true}")
+  private boolean isProd;
+
+  @Value("${FIXEDSALT_IPHASH:$2a$10$9elrbM0La5ooQgMP7i9yjO}")
```

Comment:
I think you added this default value for tests, but if the user doesn't input env variable for salt in Run/Debug, he won't receive an error while running the application. Instead, it will use this test salt. So maybe better choice is add @TestPropertySource(properties = {"FIXEDSALT_IPHASH = $2a$10$9elrbM0La5ooQgMP7i9yjO"}) to tests to fix this problem

Reply:
ok.